pipeline_steps/kibana.groovy

```diff
@@ -1,4 +1,4 @@
-def kibana(vm=null){
+def kibana(branch="master", vm=null){
```

Comment:
Given `branch` is always specified and using this function without specifying it could cause the wrong code to be tested, I think it would be better not to specify a default.